plugins/pipelines/README.md

```diff
@@ -107,10 +107,10 @@ Only PR events of specified PR number will be searched when `prNum` is set
 `GET /pipelines/{id}/events?page={pageNumber}&count={countNumber}&sort={sort}&prNum={prNumber}`
 
 #### Get all pipeline builds
-`page`, `count`, `sort`, `latest`, and `groupEventId` are optional
+`page`, `count`, `sort`, `latest`, `sortBy`, `fetchSteps`, `readOnly`, and `groupEventId` are optional
 When `latest=true` and `groupEventId` is set, only latest builds in a pipeline based on groupEventId will be returned. `latest` and `groupEventId` must be set together
 
-`GET /pipelines/{id}/builds?page={pageNumber}&count={countNumber}&sort={sort}&latest=true&groupEventId={groupEventId}`
+`GET /pipelines/{id}/builds?page={pageNumber}&count={countNumber}&sort={sort}&latest=true&groupEventId={groupEventId}&sortBy={sortBy}&fetchSteps=false&readOnly=false`
```

Comment:
`fetchSteps=false` will omit steps in the build response? This might be param that we can use to filter out the steps field in the build response.

Reply:
yes